CHANGELOG.rst

```diff
@@ -23,9 +23,8 @@ Unreleased
 * Fixed missing builtin arguments on main ``cms`` management command causing it to crash
 * Fixed template label nested translation
 * Few changes in docs/contributing/code.rst
-* Add support for Django 3.2 LTS version
```

Comment:
@goutnet Was this intended? 
Happened in your merge (develop into this branch)

Reply:
hum, no shouldn't be there, can you either remove the merge and rebase, or revert the patch?

sorry for that